mex/editor/rules/transform.py

```diff
@@ -233,20 +234,22 @@ def transform_models_to_fields(
     """
     mergeable_fields = sorted(
         {
-            f
+            (f, e.stemType)
```

Comment:
stemTyp should be the same for all extracted items and the additive item. Could you either store it in a dedicated Var or rename `mergeable_fields`, e.g. to `mergeable_fields_and_stem_types`?